java/core/src/main/java/com/microsoft/bond/Bonded.java

```diff
@@ -19,7 +19,8 @@ public Bonded(T instance) {
 
     @Override
     public void serialize(ProtocolWriter writer) throws IOException {
-        value.serialize(writer);
+        //value.serialize(writer);
```

Comment:
Simon: If you're reading this and remembering me objecting to TODOs in source, I don't particularly mind when the TODO is necessarily going to happen in the immediate future.

That definitely applies here. I think the criterion for merging this back in is that it's able to codegen, serialize, and deserialize (at least its own output from) the compat types again, and that will require us to implement this.